zerver/tests/test_realm.py

```diff
@@ -1583,7 +1583,7 @@ def test_update_realm_properties(self) -> None:
     def test_update_realm_org_type(self) -> None:
         vals = [t["id"] for t in Realm.ORG_TYPES.values()]
 
-        self.set_up_db("org_type", vals[0])
+        do_change_realm_org_type(get_realm("zulip"), vals[0], acting_user=None)
```

Comment:
As mentioned in the commit message, it seemed better to call this function instead of doing `self.assertEqual` because the initial value of this field is not the first value in the `vals` list.